drivers/mtd_spi_nor/mtd_spi_nor.c

```diff
@@ -25,24 +25,16 @@
 #include <errno.h>
 
 #include "mtd.h"
-#if MODULE_XTIMER
 #include "xtimer.h"
```

Comment:
Including `xtimer.h` without the `xtimer` module to be used will cause compilation issues on the `waspmote-pro`.

I would also keep this PR limited to cleaning up the use of `ENABLE_DEBUG`, `DEBUG()`, and custom wrapper for `DEBUG()` (such as `DEBUG_AES()`, `TRACE()`, ...). This would allow getting it in swiftly. With the high number of lines this touches, we should better get this in swiftly, to avoid having to constantly solve merge conflicts. The alignment fixes can of course also stay - changing white space should raise much worries.

Reply:
OK, I'll split this PR. All commits that replace `#if ENABLE_DEBUG` with `IS_ACTIVE(..)` will be moved to a new one.

Reply:
For next PR.